sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/AspNetCore.Benchmarks/Azure.Monitor.OpenTelemetry.AspNetCore.Benchmarks.csproj

```
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net6.0;net7.0;net462</TargetFrameworks>
```

not a blocker for me, but the CI usually requires specific frameworks provided by a specific variable.
This could break things when this repo is updated for newer frameworks.

```suggestion
    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
```

---

For local test I changed it, will revert to `$(RequiredTargetFrameworks)`